### PR TITLE
Frontend Bug Fixes

### LINKED ISSUE #30: Frontend Bug Fixes
**Describe the bug**
There are certain frontend bug fixes especially the **Rate this book** button after the last patch. All bugs are after a user signs in.

**Screenshots**

1. **Homepage** - Popular Among Users Section (Improper Button Size and Alignment)
image
2. **Your Read Section** (Improper Size, Alignment and Rating Validation)
image

**Expected behavior**
1. All buttons should be fixed as when user is not logged in
image
2. For 3. Validation should be added after a book is rated, similar to below
validation

**Definition of Done**
All sub-tasks are done and pass tests

### PR DESCRIPTION
**Pull request by reshchat**
- Changed the size and alignment of rating buttons on Homepage
- Changed the size and alignment of rating and "more details" buttons on Your read section
- Fixes #30 

## Affected Dependencies
NA

## How has this been tested?
Tested using flake8, python manage.py test

## GIF Before
CPT2104221205-1399x519

## GIF After
CPT2104221211-1491x400

## Checklist
- [X] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [X] My changes do not edit/add any unrequired files.
- [X] My changes are covered by tests.

### CODE REVIEW

**Review by maneprajakta — CHANGES_REQUESTED** [review]
- Explore page buttons should be aligned like this vertically not side by side.
Screenshot from 2021-04-23 07-24-50
- Validation is not added to `rate this book` button on your read page check home page `rate this book` button for that. 
Screenshot from 2021-04-23 07-25-41

**Comment by reshchat** [comment]
> * Explore page buttons should be aligned like this vertically not side by side.
>   Screenshot from 2021-04-23 07-24-50
> * Validation is not added to `rate this book` button on your read page check home page `rate this book` button for that.
>   Screenshot from 2021-04-23 07-25-41

@maneprajakta can you please explain what is "validation" here .

**Comment by maneprajakta** [comment]
Yep
So when I rate a book through  `rate this book` button its color should change like gif below for Your Read page.
bug2

**Comment by reshchat** [comment]
@maneprajakta I've made the necessary changes. Please check

**Review by maneprajakta — COMMENTED** [review]
- This is what we need `margin-right: -12px;` works 
Screenshot from 2021-04-23 16-17-14

**Comment by reshchat** [comment]
@maneprajakta please check

**Comment by maneprajakta** [comment]
@reshchat could you check [deployment](https://kitabe-pr-209.herokuapp.com/book_recommendations/) of your PR.
Adding only `margin-right: -12px;` on `more-details` class would have worked now it seems messed up.
If it's right at your side you can add ss of the recommendation page.

**Comment by reshchat** [comment]
Capture
Screenshot of the recommendation page.

Should I remove the margin-right: -12px; part from the rating-block
@maneprajakta

**Comment by reshchat** [comment]
@maneprajakta 
I've made some changes. Can you check it again

**Comment by maneprajakta** [comment]
> @maneprajakta
> I've made some changes. Can you check it again

@reshchat check is your screen resolution 1366 * 768.
I will put the changes in this https://github.com/Praful932/Kitabe/pull/209/commits/e3fee4fdfdfe2bf3f8c23f01f1b2ebf9aff37621 as the recent commit while be disturbing the page view on small screen.

File: static/mainapp/css/recommendation.css [hunk 1]
```diff
@@ -159,7 +160,8 @@
   padding: 8px;
   bottom: 10px;
   position: absolute;
-  left: 50%;
+  left: 45%;
+  margin-right: -12px;
```
Comment (maneprajakta) [inline static/mainapp/css/recommendation.css:163]: remove this

File: static/mainapp/css/recommendation.css [hunk 2]
```diff
@@ -159,7 +160,8 @@
   padding: 8px;
   bottom: 10px;
   position: absolute;
-  left: 50%;
+  left: 45%;
```
Comment (maneprajakta) [inline static/mainapp/css/recommendation.css]: Keep it same as early 50%

File: static/mainapp/css/recommendation.css [hunk 3]
```diff
@@ -140,10 +140,11 @@
   font-size: 15px;
   border-radius: 20px;
   position: absolute;
-  right: 26%;
+  right: 34%;
```
Comment (maneprajakta) [inline static/mainapp/css/recommendation.css]: keep same as before 26%

**Comment by reshchat** [comment]
@maneprajakta I've made the changes

**Review by maneprajakta — APPROVED** [review]
LGTM :rocket:

**Review by Praful932 — APPROVED** [review]
LGTM!

**Pull request merged**